modules/server_new/src/main/scala/observe/server/keywords/ConditionSetReader.scala

```diff
@@ -48,8 +48,9 @@ object ConditionSetReaderEpics {
         .flatMap(w => conditions.iq.map(_.toPercentile(w, Angle.fromDoubleDegrees(el))))
     )
 
-    override def cloudExtinctionStr: F[String] =
-      percentileStr(conditions.ce.map(_.toPercentile)).pure[F]
+    override def cloudExtinctionStr: F[String] = // TODOOOOOOO
```

Comment:
Is this intentional?

Reply:
Nope, forgot to re-implement it, thank you.